src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -179,6 +183,13 @@ private void ProcessProjectEvent(string messageJson)
                 });
             _plantSetter.SetPlant(projectEvent.Plant);
             _invitationRepository.UpdateProjectOnInvitations(projectEvent.ProjectName, projectEvent.Description);
+

```

Comment:
on line 72:
if (**projectEvent == null || **string.IsNullOrWhiteSpace(projectEvent.Plant) || string.IsNullOrWhiteSpace(projectEvent.ProjectName))

do the same other places where JsonSerializer.Deserialize is used in this class